StefansSuperShop/Services/MailService.cs

```diff
@@ -24,6 +26,13 @@ public MailService(IOptions<MailSettings> settings)
             _settings = settings.Value;
         }
 
+        public async Task<bool> SendContactUsAsync(MailData mailData)
```

Comment:
Behöver denna metoden ligga här, eller kan den ligga i cs filen för sidan?

Reply:
jag tror det. På något vis behöver jag få in ContactUsEmail adressen i mailData. Jag tyckte det blev kladdigt att hantera mailSettings i sidan.

Reply:
Sant...